src/users/users.service.ts

```diff
@@ -0,0 +1,53 @@
+import { Injectable } from '@nestjs/common';
+import { InjectModel } from '@nestjs/mongoose';
+import { Model } from 'mongoose';
+import { AssignRole } from './dto/assign-role.input';
+import { CreateUser } from './dto/create-user.input';
+import { UpdateUser } from './dto/update-user.input';
+import { UserDocument } from './schemas/user.schema';
+
+@Injectable()
+export class UsersService {
+  constructor(
+    @InjectModel('User') private readonly userModel: Model<UserDocument>,
+  ) {}
+
+  create(createUserInput: CreateUser) {
```

Comment:
please provide return types for eatc service method

Reply:
would you clarify the task you want me to do, please?

Reply:
Denis meant to strictly set a type of value that each method should return. 
```suggestion
  create(createUserInput: CreateUser): Promise<string> {
```
Also, isn't save an async function? You should await the result because now the ternary operation will always return a truthy value as pending Promise equal to true.

Reply:
ok, thank you I got it